.github/workflows/ci.yml

```diff
@@ -56,10 +63,69 @@ jobs:
           git config --global user.name oll-bot
           git config --global user.email developers@openlawlib.org
 
-      - name: Run pre-commit and test with pytest
+      # - name: Run pre-commit and test with pytest
+      #   run: |
+      #     pre-commit run --all-files
+      #     pytest taf/tests
+
```

Comment:
question: is running `pytest taf/tests/` and pre-commit supposed to be commented out?